system/react/src/components/Input.ts

```diff
@@ -38,7 +38,7 @@ export const baseStyles = css`
     box-shadow: 0 0 0 1px var(--border-active);
   }
 
-  &[data-error] {
+  &[data-error='true'] {
```

Comment:
maybe we should apply this pattern to `data-disabled` below?

Reply:
If we agree to use this here I think it should be changed for disabled state as well

Reply:
I think for consistency we should probably change this to use a variant name. I don't like the `='true'` in CSS selectors/html as that becomes slightly unexpected in react.

```suggestion
  &[data-variant='error'] {
```

Reply:
updated